lib/config/definitions.js

```diff
@@ -1202,6 +1202,20 @@ const options = [
     },
     mergeable: true,
   },
+  {
+    name: 'mix',
+    releaseStatus: 'alpha',
+    description: 'Configuration object for Mix module renovation',
+    stage: 'repository',
+    type: 'json',
+    default: {
+      enabled: false,
+      commitMessageTopic: 'Hex module {{depNameShort}}',
+      fileMatch: ['^mix.exs$'],
```

Comment:
Are you sure you want to only match `mix.exs` in the root of a repo? Is there zero chance of files in subdirectories? Also, replace `.` with `\\.` in this regex

Reply:
Umbrella projects have `mix.exs` files in subdirectories. 

@KonstantinKurganov it'd be lovely if umbrella projects are well handled. To my (nascent) eye, they seem like an emerging best practice to me.

Reply:
Check up umbrella projects after adding updating .lock file